config/formats.ts

```diff
@@ -2089,6 +2081,24 @@ export const Formats: FormatList = [
 		section: "Randomized Metas",
 		column: 3,
 	},
+	{
+		name: "[Gen 9] Doubles Hackmons Cup",
+
+		mod: 'gen9',
+		gameType: 'doubles',
+		team: 'randomHC',
+		searchShow: false,
+		ruleset: ['HP Percentage Mod', 'Cancel Mod'],
+		banlist: ['Nonexistent'],
+	},
+	{
+		name: "[Gen 9] Challenge Cup 2v2",
```

Comment:
Why move CC2v2 to Randomized Formats but not CC1v1?

Reply:
I figured the reason for having them in the "top slot" was to make a couple rands formats more visible while awaiting gen9, but it was a little grating to have a doubles format in the singles category, and we figured it'd be better in rands than doubles. I don't really mind either way however, just prefer cc2v2 not to be in singles.

Also, feel free to just close this as you already added DHC :)